pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
 build-backend = "hatchling.build"
-requires = [ "hatchling"]
```

Comment:
I have no idea why linting was never run on the pyproject.toml -- apologies for all these linting changes; shouldn't have been in this PR.